.changeset/shaggy-bags-work.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+Do not send `http_setLedgerOutputEnabled` messages if they reacht the HTTP Provider to prevent unwanted warnings in the local hardhat node logs
```

Comment:
typo 😅

Reply:
Good catch, thanks 🙇 

```suggestion
Do not send `http_setLedgerOutputEnabled` messages beyond the HTTP Provider to prevent unwanted warnings in the logs of the local hardhat node
```